include/gsl/span

```diff
@@ -512,23 +517,25 @@ public:
 
     constexpr reference front() const noexcept
     {
-        Expects(size() > 0);
+        Expects(size() != dynamic_extent && size() > 0);
```

Comment:
When will it ever be the case that `size() == dynamic_extent`? (Ditto line 526.)

Reply:
Shouldn't be, I'll remove those lines really quick.